go.mod

```diff
@@ -141,6 +123,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-replace github.com/ava-labs/avalanchego => github.com/chain4travel/caminogo v0.4.1
+replace github.com/ava-labs/avalanchego => github.com/chain4travel/caminogo v0.4.9-rc2.0.20230526125621-0adfa80bd7c4
```

Comment:
why not use our banff16 caminogo and caminoevm commits?

Reply:
will update as soon as the new node version is also released. We can tag the commits on these repos as well.